### ISSUE TITLE
(SEVONE) Talk's backend is off line.

### ISSUE THREAD

**Issue by denzuko** [Dallas-Makerspace/tracker#41]
## Expected Behavior 
Discuss server is operational

## Actual Behavior
502 Bad Gateway returned from server

## Steps to Reproduce the Problem

  1. visit https://talk.dallasmakerspace.org

**Comment by denzuko** [comment]
Join online meeting:
https://hello.freeconference.com/conf/call/8743701

Or, call in using your phone:
Canada: +1 867 292 3030 
United States: +1 605-562-0400 
United States: +1 218-339-7800 
United States: +1 712-832-8330 

Access code: 874 3701

One click call on your mobile:
+1 867 292 3030,,8743701#

Connection Test:
https://hello.freeconference.com/system/test

**Comment by denzuko** [comment]
team has attempted to login with provided admin key. Server is authenticating. 

Plan of action:

   - Create a new vm instance
   - Backup Volume vol-0c9031e60c190630d
   - attach a copy of the backup to the new vm instance as /recovery
   - move all configurations and code over to a private git repo

**Comment by denzuko** [comment]
Adding new relic and logentries to the new instance

**Comment by denzuko** [comment]
newrelic server monitoring is no longer availalbe and logentries is not responsive. 

Backing out of SEIM/APM install procedure.

**Comment by denzuko** [comment]
We took a different route. Dismounted the old image and mounting that into the a the instance then mounting it as /dev/sdf (shows up as /dev/vxdf inside the vm) to /mnt.

After that ran `du -hac --max-depth=4 /mnt | grep '\dG ' | sort` to find the largest storage areas.

We where able to clean up a few years worth of older kernel images in `/boot` and verify that the brooks.pem key was still in the accepted keys.

After dismounting the volume image from the recovery instance and expanding the volume by 50Gb to cover for extra growth. The EBS volume was reattached to the old instance as `/dev/sda1`. 

Once the instance came back online with the attached 150Gb EBS device the new IP address was added to cloudflare and services was quickly restored. However ssh access was not restored.

The root cause is that maintenance has not been preformed regularly and both the kernel, docker images/container,s and logs where using the available space causing the discuss container to no longer operate.

The advised course of action:

1. Backup any changes done to discourse and anything else on the instance that has changed 
2. Setup deployment via CICD
3. Include the instance in our monitoring system and tune alerting accordingly
4. Set up a second storage drive to mount at /docker for use by docker
5. Include prlxtalk in a docker swarm cluster
6. include github.com/thatjpk/cloudflare-ddns on all aws instances for automatic update of ip address changes.
